Makefile

```diff
@@ -33,17 +49,33 @@ help:
 .PHONY: all
 all: html
 
+$(OUTDIR)/inbox/%: build/inbox
```

Comment:
This can be deleted; I'm reasonably sure that this rule will never be hit if a more specific rule exists (and if one doesn't, this will do nothing so it can be removed)